tests/utils/inputparams/testparams.go

```diff
@@ -96,52 +101,56 @@ func GetEndPoint2() string {
 
 // GetSwarmManager1 returns swarm manager node IP from the configured swarm cluster
 func GetSwarmManager1() string {
-	return os.Getenv("MANAGER1")
+	manager1 := os.Getenv("MANAGER1")
```

Comment:
let's revert this change - no need to introduce a local variable here. Same for below.